lib/input_validator.rb

```diff
@@ -0,0 +1,25 @@
+require_relative "board"
+
+class InputValidator
+    def initialize
+        @invalid_moves = []
+    end
+
+    def valid_character?(move)
+        regex = /\D/ 
+        if move =~ regex || move <= 0
+            return false
+        end
+        true
+    end
+
+    def get_invalid_moves(board)
+        board.moves.each do |i|
+            @invalid_moves << i 
+        end
+    end
```

Comment:
Why would you want a list of invalid moves?

Reply:
This was my attempt to have the class (on its own) check moves that could not be requested. Ultimately this class will determine whether or not a move can be taken.

Reply:
Now that I'm looking at your last comment on this PR, I see why this way validating might cause issues.